tools/convert_tracks_csv_to_sparse_zarr.py

```diff
@@ -239,6 +239,7 @@
 points.attrs["mean_z"] = mean_z
 points.attrs["extent_xyz"] = extent_xyz
 points.attrs["fields"] = ['z', 'y', 'x', 'radius'][:num_values_per_point]
+points.attrs["ndim"] = 3 if not flag_2D else 2
```

Comment:
Nit: avoid using the negated condition when possible to help readability (less logic for the reader).

```suggestion
points.attrs["ndim"] = 2 if flag_2D else 3
```